containers/build_frontend.go

```diff
@@ -16,16 +17,19 @@ func NodeVersion(d *dagger.Client, src *dagger.Directory) *dagger.Container {
 		WithExec([]string{"cat", ".nvmrc"})
 }
 
-func CompileFrontend(d *dagger.Client, platform dagger.Platform, src *dagger.Directory, opts *YarnCacheOpts, nodeVersion string) *dagger.Directory {
+func CompileFrontend(d *dagger.Client, platform dagger.Platform, src *dagger.Directory, opts *YarnCacheOpts, version, nodeVersion string) *dagger.Directory {
 	c := NodeContainer(d, NodeImage(nodeVersion), platform).
 		WithDirectory("/src", src).
 		WithWorkdir("/src")
 
 	c = WithYarnCache(c, opts)
 
+	ersion := strings.TrimPrefix(version, "v")
```

Comment:
```suggestion
	version := strings.TrimPrefix(version, "v")
```

Reply:
lol, maybe `trimmedVersion` is better 😂

Reply:
I know that this is not a big deal but I actually think `ersion` is pretty descriptive. It does look like a typo but it more concisely describes _how_ the version is trimmed. Essentially, it is the exact same as `version` just with the `v` removed. Plus we have been using it in a lot of different places so it's probably good to keep it here for consistency.